trigger/task_jira.go

```diff
@@ -53,18 +53,55 @@ type jiraBuilder struct {
 }
 
 type jiraTemplateData struct {
-	UIRoot          string
-	Task            *task.Task
-	Build           *build.Build
-	Host            *host.Host
-	Project         *model.ProjectRef
-	Version         *version.Version
-	FailedTests     []task.TestResult
-	FailedTestNames []string
-	Tests           []jiraTestFailure
+	UIRoot             string
+	Task               *task.Task
+	Build              *build.Build
+	Host               *host.Host
+	Project            *model.ProjectRef
+	Version            *version.Version
+	FailedTests        []task.TestResult
+	FailedTestNames    []string
+	Tests              []jiraTestFailure
+	SpecificTaskStatus string
+}
+
+func makeSpecificTaskStatus(t *task.Task) string {
+	switch {
+	case t.Details.TimedOut:
+		return evergreen.TaskTimedOut
+	case t.Details.Type == evergreen.CommandTypeSystem:
+		return evergreen.TaskSystemFailed
+	case t.Details.Type == evergreen.CommandTypeSetup:
+		return evergreen.TaskSetupFailed
+	case t.Status == evergreen.TaskSucceeded:
```

Comment:
I think this needs to be above https://github.com/evergreen-ci/evergreen/pull/1572/files#diff-4d4209744bbc654e93f88a1a9fe4de6eR72, since `Details` will contain a type even in the success case.

Reply:
fixed